config.txt

        module.exports = {
            bot: {
                token: "YOUR BOT TOKEN",
                prefix: "u.",
                owners: [""],
                mongourl: "YOUR MONGO DB URL",
                servers: {
                    token: "YOUR SERVER BOT TOKE",
                    prefix: "-"
                }
            },
        
            website: {
                callback: "https://localhost/callback",
                secret: "YOUR BOTS CLIENT SECRET ",
                clientID: "", // Bot client id.
                tags: [ "Moderation", "Fun", "Minecraft","Economy","Guard","NSFW","Anime","Invite","Music","Logging", "Web Dashboard", "Reddit", "Youtube", "Twitch", "Crypto", "Leveling", "Game", "Roleplay", "Utility", "Turkish" ],
                languages: [
                    { flag: 'gb', code: 'en', name: 'English' },
                    { flag: 'tr', code: 'tr', name: 'Türkçe' },
                    { flag: 'de', code: 'de', name: 'Deutsch' }
                ],
                servers: {
                    tags: [
                    {
                        icon: "fal fa-code",
                        name: "Development"
                    },
                    {
                        icon: "fal fa-play",
                        name: "Stream"
                    },
                    {
                        icon: "fal fa-camera",
                        name: "Media"
                    },
                    {
                        icon: 'fal fa-building',
                        name: 'Company'
                    },
                    {
                        icon: 'fal fa-gamepad',
                        name: 'Game'
                    },
                    {
                        icon: 'fal fa-icons',
                        name: 'Emoji'
                    },
                    {
                        icon: 'fal fa-robot',
                        name: 'Bot List'
                    },
                    {
                        icon: 'fal fa-server',
                        name: 'Server List'
                    },
                    {
                        icon: 'fal fa-moon-stars',
                        name: 'Turkish'
                    },
                    {
                        icon: 'fab fa-discord',
                        name: 'Support'
                    },
                    {
                        icon: 'fal fa-volume',
                        name: 'Sound'
                    },
                    {
                        icon: 'fal fa-comments',
                        name: 'Chatting'
                    },
                    {
                        icon: 'fal fa-lips',
                        name: 'NSFW'
                    },
                    {
                      icon: "fal fa-comment-slash",
                      name: "Challange"
                    },
                    {
                      icon: "fal fa-hand-rock",
                      name: "Protest"
                    },
                    {
                      icon: "fal fa-headphones-alt",
                      name: "Roleplay"
                    },
                    {
                      icon: "fal fa-grin-alt",
                      name: "Meme"
                    },
                    {
                      icon: "fal fa-shopping-cart",
                      name: "Shop"
                    },
                    {
                      icon: "fal fa-desktop",
                      name: "Technology"
                    },
                    {
                      icon: "fal fa-laugh",
                      name: "Fun"
                    },
                    {
                      icon: "fal fa-share-alt",
                      name: "Social"
                    },
                    {
                      icon: "fal fa-laptop",
                      name: "E-Spor"
                    },
                    {
                      icon: 'fal fa-palette',
                      name: 'Design'
                    },
                    {
                      icon: 'fal fa-users',
                      name: 'Community'
                    }
                    ]                
                }
            },
        
            server: {
                id: "SERVER ID",
                invite: "INVITE",
                roles: {
                    administrator: "847365325594820618",
                    moderator: "849082710398992416",
                    profile: {
                        sitecreator : "849786131612565514",
                        booster: "853182672356245506",
                        sponsor: "849083996339175424",
                        supporter: "849083653635964959",
                        partnerRole: "849084095493570601"
                    },
                    codeshare: {
                        javascript: "",
                        html: "",
                        substructure: "",
                        bdfd: "", // Bot Designer For Discord
                        fiveInvite: "",
                        tenInvite: "",
                        fifteenInvite: "",
                        twentyInvite: ""
                    },
                    botlist: {
                        developer: "847730047955632149",
                        certified_developer: "849082176573669396",
                        bot: "847364865567359006",
                        certified_bot: "849082418057314334",
                    }
                },
                channels: {
                    codelog: "849128561897635880",
                    login: "849128561897635880",
                    webstatus: "849128561897635880",
                    uptimelog: "849128561897635880",
                    botlog: "849079385423872021",
                    votes: "849074702822801478"
                }
            }
        
        
        }


config.js

        module.exports = {
            bot: {
                token: "Nzk0MDM5ODkwNTA1MjM2NTMw.X-1BRg.WhOPJ6fhfGpoYXCrR-M3OkSU7Og",
                prefix: "u.",
                owners: ["781612763566964786" ,"605994817918599179" ,"321590712665636865", "815204465937481749"],
                mongourl: "mongodb+srv://TestBot:X-wing89@botnox.xrnpl.mongodb.net/upcord?retryWrites=true&w=majority",
                servers: {
                    token: "ODU5NTk5NzIxMjM2Mzk4MTAw.YNvCoA.7nvnzw8NWj_YVQ0KPiWmkbmB89I",
                    prefix: "-"
                }
            },
        
            website: {
                callback: "https://upcord.xyz/callback",
                secret: "UWkMowmUfMRgQfFDi6GAatTmRuBrPQPU",
                clientID: "794039890505236530", // Bot client id.
                tags: [ "Moderation", "Fun", "Minecraft","Economy","Guard","NSFW","Anime","Invite","Music","Logging", "Web Dashboard", "Reddit", "Youtube", "Twitch", "Crypto", "Leveling", "Game", "Roleplay", "Utility", "Turkish" ],
                reporttags: ["Choose one...","NSFW content","API abuse","Malicious use of bot page","Copycat","Doesn't work","Other"],
                languages: [
                    { flag: 'gb', code: 'en', name: 'English' },
                    { flag: 'tr', code: 'tr', name: 'हिंदी' },
                    { flag: 'de', code: 'de', name: 'Deutsch' },
                    { flag: 'it', code: 'it', name: 'Italiano' },
                    { flag: 'np', code: 'np', name: 'नेपाली' },
                    { flag: 'id', code: 'id', name: 'indonesia' },
                    { flag: 'ar', code: 'ar', name: 'العربية' },
                    { flag: 'fr', code: 'fr', name: 'French' },
                    { flag: 'ae', code: 'ae', name: 'عربي' },
                    { flag: 'pl', code: 'pl', name: 'Polish' }
                ],
                servers: {
                    tags: [
                    {
                        icon: "fal fa-code",
                        name: "Development"
                    },
                    {
                        icon: "fal fa-play",
                        name: "Stream"
                    },
                    {
                        icon: "fal fa-camera",
                        name: "Media"
                    },
                    {
                        icon: 'fal fa-building',
                        name: 'Company'
                    },
                    {
                        icon: 'fal fa-gamepad',
                        name: 'Game'
                    },
                    {
                        icon: 'fal fa-icons',
                        name: 'Emoji'
                    },
                    {
                        icon: 'fal fa-robot',
                        name: 'Bot List'
                    },
                    {
                        icon: 'fal fa-server',
                        name: 'Server List'
                    },
                    {
                        icon: 'fal fa-moon-stars',
                        name: 'Turkish'
                    },
                    {
                        icon: 'fab fa-discord',
                        name: 'Support'
                    },
                    {
                        icon: 'fal fa-volume',
                        name: 'Sound'
                    },
                    {
                        icon: 'fal fa-comments',
                        name: 'Chatting'
                    },
                    {
                        icon: 'fal fa-lips',
                        name: 'NSFW'
                    },
                    {
                      icon: "fal fa-comment-slash",
                      name: "Challange"
                    },
                    {
                      icon: "fal fa-hand-rock",
                      name: "Protest"
                    },
                    {
                      icon: "fal fa-headphones-alt",
                      name: "Roleplay"
                    },
                    {
                      icon: "fal fa-grin-alt",
                      name: "Meme"
                    },
                    {
                      icon: "fal fa-shopping-cart",
                      name: "Shop"
                    },
                    {
                      icon: "fal fa-desktop",
                      name: "Technology"
                    },
                    {
                      icon: "fal fa-laugh",
                      name: "Fun"
                    },
                    {
                      icon: "fal fa-share-alt",
                      name: "Social"
                    },
                    {
                      icon: "fal fa-laptop",
                      name: "E-Spor"
                    },
                    {
                      icon: 'fal fa-palette',
                      name: 'Design'
                    },
                    {
                      icon: 'fal fa-users',
                      name: 'Community'
                    }
                    ]                
                }
            },
        
            server: {
                id: "847364462876033064",
                id2: "871928445209309184",
                invite: "https://discord.gg/3JJ5HRG3cp",
                roles: {
                    administrator: "847365325594820618",
                    bh: "877411998281912421",
                    bhg: "877412091408027709",
                    yonetici: "849786131612565514",
                    community: "847730047955632149",
                    manager: "873473305993162753",
                    ofbots: "849154041032343572",
                    moderator: "849082710398992416",
                    sitecreator : "871923370449571840",
                    premium_developer: "853182672356245506",
                    moderatorrole: "877449159504973846",
                        booster: "853182672356245506",
                        sponsor: "849083996339175424",
                        supporter: "849083653635964959",
                        partnerRole: "849084095493570601",
                    profile: {
                        sitecreator : "871923370449571840",
                        booster: "853182672356245506",
                        sponsor: "849083996339175424",
                        supporter: "849083653635964959",
                        partnerRole: "849084095493570601"
                    },
                    codeshare: {
                        javascript: "849798525281894431",
                        html: "849803601702420511",
                        substructure: "849803796392574996",
                        bdfd: "849803937157611522", // Bot Designer For Discord
                        fiveInvite: "",
                        tenInvite: "",
                        fifteenInvite: "",
                        twentyInvite: ""
                    },
                    botlist: {
                        developer: "847730047955632149",
                        ownerserver: "",
                        boosted_bot: "871957389400743967", // Boosted Bot Role ID
                        promoted_bot: "876717829070667857",
                        premium_developer: "876718030074302528",
                        promoted_developer: "876718397142990899",
                        boosted_developer: "853182672356245506",
                        certified_developer: "849082176573669396",
                        bot: "847364865567359006",
                        certified_bot: "849082418057314334",
                    }
                },
                channels: {
                    codelog: "849128561897635880",
                    reportlog: "873463903923871755",
                    login: "849128561897635880",
                    webstatus: "849128561897635880",
                    uptimelog: "849128561897635880",
                    botlog: "849079385423872021",
                    votes: "849074702822801478"
                }
            }
        
        
        }